src/components/interactive-builder/add-question.modal.tsx

```diff
@@ -495,6 +503,29 @@ const AddQuestionModal: React.FC<AddQuestionModalProps> = ({
                       onChange={(event: React.ChangeEvent<HTMLInputElement>) => setRows(event.target.value)}
                       required
                     />
+                  ) : renderingType === 'toggle' ? (
+                    <div>
+                      <TextInput
+                        id="lableTrue"
```

Comment:
Please check for the typos everywhere else as well,
```suggestion
                        id="labelTrue"
```